<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vldrChatQtClass</class>
 <widget class="QMainWindow" name="vldrChatQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>340</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>vldrChatQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">
QWidget {
background-image: url(&quot;:/vldrChatQt/cartographer.png&quot;);
    background-repeat: repeat-y;
	
}</string>
   </property>
   <widget class="QListWidget" name="usersList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>271</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {

border:1px solid rgb(27, 27, 27);
background:none;
background-color:rgb(43, 43, 43);
color:white;
border-top-left-radius: 3px;
border-bottom-left-radius: 3px;
}</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="chatBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>331</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
border:1px solid rgb(27, 27, 27);
background:none;
background-color:rgb(43, 43, 43);
color:white;
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(27, 27, 27);
background:none;
background-color:rgb(43, 43, 43);
color:white;
padding:5px;
font-size:18px;
border-radius:2px;</string>
    </property>
    <property name="placeholderText">
     <string>Say something nice!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid rgb(27, 27, 27);
background:none;
background-color:rgb(43, 43, 43);
color:rgb(173, 173, 173);
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
}

QPushButton:hover {
	color:white;
}

QPushButton:pressed {
	border-bottom:0px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top:2px solid rgb(0, 246, 122);
background:none;</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>usersList</zorder>
   <zorder>chatBox</zorder>
   <zorder>messageBox</zorder>
   <zorder>sendButton</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="vldrChatQt.qrc"/>
 </resources>
 <connections/>
</ui>
